roles/aap_ocp_install/tasks/pre-validate.yml

```
@@ -94,23 +94,43 @@
   when:
     - ( 'operator' in ansible_run_tags ) or ( 'all' in ansible_run_tags and aap_ocp_install_operator is defined )
 
+- name: Retrieve the requested operator channel version numbers
+  ansible.builtin.set_fact:
+    __aap_ocp_install_major_version: "{{ aap_ocp_install_operator['channel'] | regex_search(__aap_ocp_install_major_version_re, '\\1') | first }}"
+    __aap_ocp_install_minor_version: "{{ aap_ocp_install_operator['channel'] | regex_search(__aap_ocp_install_minor_version_re, '\\1') | first }}"
+  vars:
+    __aap_ocp_install_major_version_re: '(\d+)\.\d+'
+    __aap_ocp_install_minor_version_re: '\d+\.(\d+)'
+
+- name: Set a flag indicating whether to use the new (AAP 2.5+) operator installation method
+  ansible.builtin.set_fact:
+    __aap_ocp_install_25_install: "{{ (__aap_ocp_install_major_version | int) > 2 \
+                                  or (( __aap_ocp_install_major_version | int) == 2 \
+                                  and ( __aap_ocp_install_minor_version | int) >= 5) }}"
```

To fix an ansible-lint warning:

```suggestion
                                  and (__aap_ocp_install_minor_version | int) >= 5) }}"
```